step/createbackup/lang/en/lifecyclestep_createbackup.php

```diff
@@ -24,3 +24,5 @@
  */
 
 $string['pluginname'] = 'Create Backup Step';
+
+$string['createbackup_maximumbackupspercron'] = 'Maximum number of Backups per cron';
```

Comment:
I know that the deletecourse step is doing it differently, but the 'createbackup_' prefix is obsolete here i think.